cli/app.go

```diff
@@ -43,15 +43,21 @@ type appServices interface {
 	repositoryWriterAction(act func(ctx context.Context, rep repo.RepositoryWriter) error) func(ctx *kingpin.ParseContext) error
 	maybeRepositoryAction(act func(ctx context.Context, rep repo.Repository) error, mode repositoryAccessMode) func(ctx *kingpin.ParseContext) error
 
+	repositoryConfigFileName() string
+	getProgress() *cliProgress
+}
+
+type coreAppServices interface {
```

Comment:
_core_ suggests that it's the basic building block and thus a subset (and not a superset) of `appServices`

Reply:
renamed to `advancedAppServices`